src/atoms/Layout/utils.js

```diff
@@ -8,15 +8,15 @@ export function processChild(child, layoutProps) {
   // TODO: Find a better way to type check here.
   // I believe the first now fails because of Preact
   if (
-    child.type === Col ||
-    get(child, 'nodeName.prototype.displayName') === 'Col' ||
-    get(child, 'type.prototype.displayName') === 'Col' ||
-    get(child, 'type.name') === 'Col' ||
-    get(child, 'type.rclName') === 'Col' ||
+    child.type === Col
+    || get(child, 'nodeName.prototype.displayName') === 'Col'
```

Comment:
what was the linter error that caused this refactor? the previous style makes more sense IMO

Reply:
This was done automatically by the linter at some point while we were trying to update it. I hate it so I've changed it back to the way it was and it's working fine.